src/Components/ArticleById.jsx

```diff
@@ -21,11 +24,15 @@ export default function ArticleById() {
       setArticlesCom(getArticlesWithComments);
       setLoading(true);
```

Comment:
these should be separated out into separate components rather than your ArticleById component trying to do everything, this is leading to errors like not being able to get your votesNo state set to the correct information from the start
We should have a component for:
 Single article (ArticleById)
 Article comments
 Patch votes